src/main/Graph.java

```diff
@@ -177,6 +205,11 @@ public String shortestPath(Vertex<?> v1, Vertex<?> v2) {
 	@Override
 	public String MST() {
 		// TODO @Andre
+		
+		// Edge<?>[] edges = new Edge<?>[this.getEdgeNumber()];
+		
+		// precisa de acesso a lista de adjacencia.

```

Comment:
Each vertex has an adjacency list. So, you have access to this.

Reply:
Unfortunatly, i have no access to any vertex unless given as a parameter. I do have access to a vertex Set, but access to a Set requires a reference.